FAQ/FAQ-LightningNetwork.md

```diff
@@ -318,6 +318,59 @@ cd btcpayserver-docker
 export LIGHTNING_ALIAS="namehere"
 . ./btcpay-setup.sh -i
 ```
+### How to back up LND channels offsite
+https://github.com/btcpayserver/btcpayserver-doc/blob/master/FAQ/readme.md
```

Comment:
Thanks for the quick changes.

Sorry if I was unclear, but actually what you need to do is edit the file  https://github.com/btcpayserver/btcpayserver-doc/blob/master/FAQ/readme.md  and add a link there to this document, you can remove this link from here.
So what you need to do is add a link like 
[How to back up LND channels offsite?](FAQ-LightningNetwork.md#how-to-back-up-lnd-channels-offsite)
![Screenshot 2020-02-21 at 08 44 27](https://user-images.githubusercontent.com/36959754/75014366-63817200-5486-11ea-8e41-72fb29c1a995.png) 
into that readme file in /FAQ
Check how it's done for other questions, if it's still not quite clear I can add it for you
```suggestion

```

Reply:
my bad, sorry, this is my first PR)
Hope I got it right this time